### PR TITLE
emulator clarity

### PR DESCRIPTION
**Pull request by gundralaa**
- [x] changed incorrect docs
- [x] saved uart output to text file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,14 +13,22 @@ See the [getting started instructions](https://www.lf-lang.org/embedded-lab/Gett
...
 Because it emulates the Pico board, not the Pololu robot, it will not be able to run all the programs for the robot.
 
-By default, the emulator uses **hex** binaries which are generated by both build options. Any hex files that are in need of testing must be placed in the ``/target/hex/`` directory.
+By default, the emulator uses **hex** binaries which are generated by both build options. Any hex files that are in need of testing must be placed in the ``/test`` directory.
 
-Run the following from the ``/test/`` directory. It will run an emulator instance for each hex file in the ``hex`` directory in parallel and report results as plain text files.
+Run the following from the ``/test`` directory. It will run an emulator instance for each hex file in the directory in parallel and report results as plain text files in the same directory.
```
Comment (edwardalee) [inline README.md:18]: I just tried this and HelloPico.hex fails, but no text files are created.

**Review by edwardalee — APPROVED** [review]
This looks like an improvement. Let's merge it.  I include a question in my review.

File: README.md (hunk 1 above)
Comment (gundralaa) [reply README.md:18]: Is this still an issue? I'm seeing a txt output in the form of "HelloPico.hex_out.txt" in the test folder on failed tests.

Comment (edwardalee) [reply README.md:18]: Yes, this seems to work now.

**Pull request merged**